src/main/java/org/alfresco/rest/api/impl/GroupsImpl.java

```diff
@@ -367,7 +395,7 @@ else if (zonesParam.isEmpty())
                 // Get authorities using canned query but without using
                 // the requested paginating now because we need to filter out
                 // the root authorities.
-                PagingResults<AuthorityInfo> nonPagingResult = authorityService.getAuthoritiesInfo(authorityType, zoneFilter, null, sortProp.getFirst(), sortProp.getSecond(),
+                PagingResults<AuthorityInfo> nonPagingResult = authorityService.getAuthoritiesInfo(authorityType, zoneFilter, groupNamesFilter, sortProp.getFirst(), sortProp.getSecond(),

```

Comment:
I think we are missing the null as authority

Reply:
I use that value to pass diplayName to the canned query